test/unit/remote_tests.rb

```diff
@@ -35,6 +37,13 @@ class UnitTests < Assert::Context
     end
     subject{ @cmd }
 
+    private
+
+    def ssh_cmd_str(cmd_str, host, args)
+      val = "\"#{cmd_str}\"".gsub("\\", "\\\\\\").gsub('"', '\"')
+      "ssh #{args} #{host} -- \"sh -c #{val}\""
+    end
```

Comment:
@kellyredding - eh, it's not ideal but its kinda like testing sql. You can test the sql strings but that doesn't prove it works. I see this as more of a way to notify us of regressions or other changes. Essentially, if the ssh cmd string changes we need to be aware.

Later on, if we want to add system tests that actually do a remote command then I'd be ok with that to help verify everything works as expected. There are a lot of issues with having a machine to SSH into but assuming we figure out a good way to handle those it'd be nice.